stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/internal/MessageInputFieldView.kt

```diff
@@ -61,16 +62,29 @@ internal class MessageInputFieldView : FrameLayout {
     internal val selectedAttachmentsCount: StateFlow<Int> = _selectedAttachmentsCount
 
     var mode: Mode by Delegates.observable(Mode.MessageMode) { _, oldMode, newMode ->
-        if (oldMode != newMode) onModeChanged(newMode)
+        if (oldMode != newMode) {
+            onModeChanged(newMode)
+        }
+    }
+
+    private fun modeChangeIsAllowed(oldMode: Mode, newMode: Mode): Boolean {
+        return if (oldMode is Mode.EditMessageMode && newMode is Mode.CommandMode) {
+            false.also {
+                Toast.makeText(context, "It is not possible to use a command when editing messages", Toast.LENGTH_SHORT)
```

Comment:
I imagine we should be using a translation here?

I would also just toast and then return false, than use .also {}, just seems simpler!

Reply:
I removed the translations and now it is not possible to see the pop up with commands suggestions now. The messages ends up being to the developer in case he/she changes the UI to show commands anyways.